compact/definitions.xml

```diff
@@ -514,17 +515,18 @@ Service gaps in FW direction (before endcapP ECAL) and BW direction (before endc
     <comment> extra 50cm rmax that "protrudes" into the HCAL</comment>
     <constant name="EcalEndcapP_rmax"               value="floor(Eta1_1_tan * EcalEndcapP_zmin)"/>
 
-    <constant name="EcalEndcapN_zmin"               value="BackwardPIDRegion_zmin + BackwardInnerEndcapRegion_length"/>
-    <constant name="EcalEndcapN_length"             value="60*cm" />
+    <!-- <constant name="EcalEndcapN_zmin"               value="BackwardPIDRegion_zmin + BackwardInnerEndcapRegion_length"/> -->
+    <constant name="EcalEndcapN_zmin"               value="174*cm"/>   <!-- Currently fix value hardcoded -->
```

Comment:
This is also incorrect compared to https://eic.jlab.org/Geometry/Detector/Detector-20220929172703.html

Reply:
OK, it supposed to be -174 cm.... I somehow know why I can't pass the overlap check on github but in local

Reply:
Ah, sorry, you're right, I misread. I thought you had 170 somewhere.